### PR TITLE
chore(components): refine checkbox style

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/components/src/lib/checkbox/checkbox.scss [hunk 1]
```diff
@@ -34,11 +36,19 @@ $gutter: 4px;
 	@supports (user-select: none) {
 		user-select: none;
 	}
+
+	&:not(.disabled) {
```
Comment (rachelbt) [inline libs/components/src/lib/checkbox/checkbox.scss:40]: what about readonly?
```suggestion
	&:not(.disabled, .readonly) {
```

File: libs/components/src/lib/checkbox/checkbox.scss [hunk 2]
```diff
@@ -25,6 +25,8 @@ $gutter: 4px;
 	display: inline-flex;
 	align-items: center;
 	gap: 12px;
```
Comment (rachelbt) [inline libs/components/src/lib/checkbox/checkbox.scss:27]: I know this is not related to the PR but if radio is 8px gap (or 10px) that shouldn't it be the same here?

File: libs/components/src/lib/checkbox/checkbox.scss (hunk 1 above)
Comment (yinonov) [reply libs/components/src/lib/checkbox/checkbox.scss:40]: Readonly is still interactive

File: libs/components/src/lib/checkbox/checkbox.scss (hunk 2 above)
Comment (yinonov) [reply libs/components/src/lib/checkbox/checkbox.scss:27]: it's addressed at https://github.com/Vonage/vivid-3/pull/697/files#diff-aa6257cff9e49a629c169c30f42ecac73cfa91357575d9cf13511ac0336eff05R28

**Review by rachelbt — APPROVED** [review]
LGTM

**Pull request merged**